dlp/api/DlpSample/DeIdentify.cs

```diff
@@ -1,49 +1,65 @@
-﻿using Google.Cloud.Dlp.V2;
+﻿// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Dlp.V2;
 using Google.Protobuf;
 using System;
 using System.Collections.Generic;
 using System.IO;
+using static Google.Cloud.Dlp.V2.CryptoReplaceFfxFpeConfig.Types;
 
 namespace GoogleCloudSamples
 {
     public class DeIdentify : DlpSampleBase
     {
         // [START dlp_deidentify_masking]
         public static object DeidMask(
-            string projectId,
-            string value,
-            string infoTypes,
-            string maskingCharacter,
-            int numberToMask,
-            bool reverseOrder)
+            string ProjectId,
```

Comment:
Please, revert. Input parameters should start with small letter, not capital. Capital letter is for properties and constants that typically are public to the class.